packages/driver/src/cypress/runner.ts

```diff
@@ -1541,6 +1541,7 @@ export default {
           }
 
           cy.state('duringUserTestExecution', false)
```

Comment:
prob an enhancement for later, but would it make sense for cy.state to manage when to push to the primary Origin Communicator when a state value is updated instead of adding it each place we want to use it?

Reply:
We considered that, but it's not really necessary, since Cypress commands can't run in parallel, there's no need for the syncing to be real-time. It's enough to sync in when cy.origin starts and sync out when it's finished.